less/availity-cards.less

```diff
@@ -63,6 +91,29 @@
   }
 }
 
+.card-header-link {
```

Comment:
My assumption is this will be used within .card-title. If so, can we use .card-title-link instead of header? We need consistent terminology.